editor/property_selector.cpp

```diff
@@ -569,5 +569,7 @@ PropertySelector::PropertySelector() {
 
 	help_bit = memnew(EditorHelpBit);
 	vbc->add_margin_child(TTR("Description:"), help_bit);
+	help_bit->get_rich_text()->set_fit_content(false);
+	help_bit->get_rich_text()->set_custom_minimum_size(Size2(help_bit->get_rich_text()->get_minimum_size().x, 135));
```

Comment:
Scale to hiDPI displays:
```suggestion
help_bit->get_rich_text()->set_custom_minimum_size(Size2(help_bit->get_rich_text()->get_minimum_size().x, 135 * EDSCALE));
```

Reply:
I am new to github (this is my first PR ever). do I need to do anything with this?

Reply:
Please make the change to your local copy, stage your changes, then force-push an amended commit as described in https://docs.godotengine.org/en/stable/contributing/workflow/pr_workflow.html#the-interactive-rebase.

Reply:
I made the changes